### ISSUE TITLE
Packaged game don't start Node.JS server

### ISSUE THREAD

**Issue by Ordrinn11** [getnamo/NodeJs-Unreal#47]
Hi everyone, 

My server is working perfectly on editor but did not start on a packaged game.
I have not much information but i can see that the node process do not start on game launch.

Thanks for your work !

**Comment by getnamo** [comment]
Might need to see some logs to see what it spits out when it starts. Make sure your scripts are placed as expected in https://github.com/getnamo/NodeJs-Unreal#packaging

**Comment by Ordrinn11** [comment]
Of course, this is the logs for the package : [logspackagenode.txt](https://github.com/getnamo/NodeJs-Unreal/files/10007998/logspackagenode.txt)
